ddp-workspace/projects/ddp-dsm-ui/src/app/utils/utils.ts

```diff
@@ -662,7 +662,7 @@ export class Utils {
     }
     let text = answer;
     let ans;
-    if (qdef.groups) {
+    if (qdef.groups.length > 0) {
```

Comment:
If it's possible for `qdef.groups` to be `null`/`undefined` then it's gonna throw an exception. Maybe `qdef.groups?.length`?